sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -46,5 +46,14 @@ internal TextAnalyticsRequestOptions(bool includeStatistics, string modelVersion
         /// <see cref="StringIndexType.Utf16CodeUnit"/>.
         /// </summary>
         public StringIndexType StringIndexType { get; set; } = StringIndexType.Utf16CodeUnit;
+
+        /// <summary>
+        /// If set to true (the default for `StartAnalyzeHealthcareEntities` and `RecognizePiiEntities`),
+        /// you opt-out of having your text input logged for troubleshooting. By default, Text Analytics
+        /// logs your input text for 48 hours, solely to allow for troubleshooting issues.
+        /// Setting this parameter to true, disables input logging and may limit our ability
+        /// to investigate issues that occur.
+        /// </summary>
```

Comment:
Please add a comment to explain what is the default value. The default is true for pii and health and false otherwise.

Reply:
plop. I copied this from JS

Reply:
ops I missed that part. This looks good :)

Reply:
updated